pq-crypto/bike_r3/measurements.h

```diff
@@ -0,0 +1,87 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file is not used so we can remove it.

Reply:
Fixed